birdhouse/config/geoserver/default.env

```diff
@@ -26,9 +26,14 @@ export GEOSERVER_COMMUNITY_EXTENSIONS="geopkg-plugin"
 # Must use single-quote for delayed eval.
 export GEOSERVER_DATA_DIR='${DATA_PERSIST_ROOT}/geoserver'
 
+# If set, requests to the geoserver endpoint will not be authorized through twitcher/magpie
+export GEOSERVER_SKIP_AUTH=False
+export GEOSERVER_SKIP_AUTH_PROXY_INCLUDE='$([ x"${GEOSERVER_SKIP_AUTH}" = x"True" ] && echo "return 200;")'
```

Comment:
@mishaschwartz Thanks for adding `GEOSERVER_SKIP_AUTH` this PR looks good for me as well.  

Can you eventually add a similar switch for THREDDS.  Currently we are applying a custom hack on our fork of this repo for production to disable Twitcher in front of our Thredds.  Performance increased between 3 and 4x.

@fmigneault is there a DB upgrade?  If no upgrade I won't need to deploy this PR to test and can approuve straight.

Reply:
No DB upgrade in Magpie. The existing `geoserser` type is extended.